src/test/java/homework/task_6_exceptions/ArrayValueCalculatorTest.java

```diff
@@ -0,0 +1,68 @@
+package homework.task_6_exceptions;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.*;
+
+
+class ArrayValueCalculatorTest {
+    //1
+    @Test
+    void testValidInputArray() {
+        String[][] arr = {
+                {"1", "2", "3", "4" },
+                {"5", "6", "7", "8" },
+                {"9", "10", "11", "12" },
+                {"13", "14", "15", "16"}
+        };
+        ArrayValueCalculator arrCalc = new ArrayValueCalculator();
+        int result = arrCalc.doCalc(arr);
+        assertEquals(136, result);
+    }
+
+    // Тест 2 зробив за допомогою ChatGPT, бо не розумів як обробляти винятки
+
+    //fail() - це метод з бібліотеки JUnit, який викликається в unit-тестах, коли очікується, що тест не пройде успішно.
+    // Виклик fail() означає, що у теста виявлено невідповідність або помилку, і тест вважається проваленим.
+
+    //2
+    @Test
+    void testInvalidInputArray() {
+        String[][] arr = {
+                {"1", "2", "3", "4" },
+                {"1", "2", "3", "4" },
+                {"5", "6", "7", "8" },
+                {"9", "10", "11", "12" },
```

Comment:
додай ще один тест, в якому ось цей рядок буде виглядати ось так: 
```suggestion
                {"9", "10", "11", "12", "13" },
```
має бути виключення, а у тебе не буде

Reply:
Тест (2.1) згідно Вашего зауваження додано.